blosc/shuffle-altivec.c

```diff
@@ -579,6 +725,17 @@ unshuffle_altivec(const int32_t bytesoftype, const int32_t blocksize,
     return;
   }
 
+  /* Warning in case of unaligned buffers*/
+  if (((size_t)_dest) % 16)
+  {
+       printf("shuffle2_altivec: Unaligned destination !\n");
```

Comment:
Be consistent and use 2 spaces for indentation (the convention for Blosc2 code).

Reply:
I'll setup the editor accordingly. thanks